src/music.c

```diff
@@ -139,7 +139,7 @@ calm_nymphs(int distance)
             continue;
         if (mtmp->data->mlet == S_NYMPH && mtmp->mcanmove
             && distu(mtmp->mx, mtmp->my) < distance) {
-            wakeup(mtmp, FALSE);
+            wakeup(mtmp, FALSE, FALSE);
```

Comment:
Not that this would be called for mimics, but I feel like a pacified mimic would come out of hiding?